src/__forks__/interface/RelayConnectionInterface.js

```diff
@@ -80,4 +80,4 @@ var RelayConnectionInterface = {
   },
 };
 
-module.exports = RelayConnectionInterface;
\ No newline at end of file
+module.exports = RelayConnectionInterface;
```

Comment:
We cannot accept this PR if there are changes to the `src/` directory - rules should lenient enough to allow the code to pass as-is.